src/redux/contractors/contractorsReducer.js

```diff
@@ -0,0 +1,38 @@
+import {
+    SET_ERRORS,
+    GET_CONTRACTORS,
+    ADD_CONTRACTOR
+} from './types'
+
+const initialState = {
+    loading: false,
+    error: null,
+    contractorsList: null
```

Comment:
for arrays I would consider using `[]` instead of null. Null is good for objects. For arrays you can use empty array so all you .map, .filter, etc methods won't crash. 
Sometimes you will need to change if array is empty, then most common solution is to use isEmpty method from lodash library

```js
import { isEmpty } from 'lodash'

const foo = []
const bar = [1,2,3]

isEmpty(foo) // true
isEmpty(bar) // false
```